pom.xml

```diff
@@ -21,15 +22,21 @@
 		<maven.javadoc.plugin>2.10.3</maven.javadoc.plugin>
 		<java.compiler.plugin.version>2.5.1</java.compiler.plugin.version>
 		<maven.deploy.plugin.version>2.8.2</maven.deploy.plugin.version>
-		<maven.release.plugin.version>2.5.2</maven.release.plugin.version>
+		<maven.release.plugin.version>2.5.3</maven.release.plugin.version>
 		<junit.version>4.12</junit.version>
 		<slf4j.version>1.7.12</slf4j.version>
+		<maven-dependency-plugin.version>2.4</maven-dependency-plugin.version>
+		<maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+		<maven-assembly-plugin.version>2.6</maven-assembly-plugin.version>
+		<findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
+		<maven-plugin-plugin.version>3.2</maven-plugin-plugin.version>
+		<maven-project-info-reports-plugin.version>2.8.1</maven-project-info-reports-plugin.version>
 	</properties>
 
 	<scm>
-		<url>http://git.dev.afrigis.co.za/components/afrigis-services</url>
-		<connection>scm:git:ssh://git@git.dev.afrigis.co.za/components/afrigis-services.git</connection>
-		<developerConnection>scm:git:ssh://git@git.dev.afrigis.co.za/components/afrigis-services.git</developerConnection>
+		<url>https://github.com/AfriGIS-South-Africa/afrigis-services-java</url>
```

Comment:
Changes based on 

- https://maven.apache.org/scm/git.html 
- http://blog.sonatype.com/2009/09/maven-tips-and-tricks-using-github/

Not tested yet.